### PR TITLE
Refactor Name To Title

### PR DESCRIPTION
**Pull request by bryanwongweiheng**

### CODE REVIEW

**Comment by pangjiahao** [comment]
This is the one where manual testing all okay but travis check fail?

**Comment by bryanwongweiheng** [comment]
travis was failing before this PR. So i dont think the problem originates here

File: src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import seedu.address.ui.TestFxmlObject?>
```
Comment (vig42) [inline src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml]: What does this import do?

Comment (bryanwongweiheng) [reply src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml]: not sure. but i found it in the team repo and it was missing from my own. so i just put it back there

Comment (bryanwongweiheng) [reply src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml]: what's probably happening is that the class / object that is being made from this fxml file requires that import

File: src/test/data/XmlSchedulerStorageTest/invalidCalendarEventScheduler.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <scheduler>
     <!-- CalendarEvent with invalid name field -->
     <calendarEvents>
-        <name>Ha!ns Mu@ster</name>
+        <name> Ha!ns Mu@ster</name>
```
[the remark this answers is not in the document]
Comment (bryanwongweiheng) [reply src/test/data/XmlSchedulerStorageTest/invalidCalendarEventScheduler.xml]: actually it's gonna be title. yeah. What would you consider an invalid title? I would consider non-alphanumeric characters as valid in a title.

[the remark this answers is not in the document]
Comment (bryanwongweiheng) [reply src/test/data/XmlSchedulerStorageTest/invalidCalendarEventScheduler.xml]: anyway i didn't change the xml files unless they were failing the tests. This commit was just to change mostly what was in the model

[the remark this answers is not in the document]
Comment (vig42) [reply src/test/data/XmlSchedulerStorageTest/invalidCalendarEventScheduler.xml]: > 
> 
> actually it's gonna be title. yeah. What would you consider an invalid title? I would consider non-alphanumeric characters as valid in a title.

Agreed. I guess we shouldn't allow a title with nothing but whitespace in it, but anything else should be ok.

For some of the other tests (e.g. CommandTestUtil, ParserUtilTest), you've changed the example of an invalid title to be just a blank string " ", so maybe could do that here as well?

> 
> 
> anyway i didn't change the xml files unless they were causing some tests to fail. This commit's aim was to change what was in the model first, and change its tests. Any changes in the other components for those other tests that were failing

Ah yeah, that makes sense.

[the remark this answers is not in the document]
Comment (bryanwongweiheng) [reply src/test/data/XmlSchedulerStorageTest/invalidCalendarEventScheduler.xml]: ok

[the remark this answers is not in the document]
Comment (pangjiahao) [reply src/test/data/XmlSchedulerStorageTest/invalidCalendarEventScheduler.xml]: I think we can allow the title to be almost anything, if we have the prefix thing, like n/ or p/

**Pull request merged**